src/server/headed-mode-plugin.js

```diff
@@ -1,16 +1,17 @@
 import { globSync } from 'glob';
 
-export function headedMode({ manual, watch, pattern }) {
+export function headedMode({ open, watch, pattern }) {
 
 	const files = globSync(pattern, { ignore: 'node_modules/**', posix: true });
 
+	const delay = ms => ms && new Promise(r => setTimeout(r, ms));
+
 	return {
 		name: 'brightspace-headed-mode',
 		async transform(context) {
-			if ((watch || manual) && files.includes(context.path.slice(1))) {
-				// allow time to open devtools in firefox and webkit
-				watch && await new Promise(r => setTimeout(r, 2000));
-				return `debugger;\n${context.body}`;
+			if ((watch || open) && files.includes(context.path.slice(1))) {
+				await delay(0);
+				return `debugger;\nimport '@brightspace-ui/testing/pause.js';\n${context.body}`;
```

Comment:
Do we still need to pause the debugger now that there's a "start" button?

Reply:
I think so. Pausing gives you time to open the devtools, and then starting will plop you right on the test file because of `debugger;` where you can then set other breakpoints. Also useful because the browser has no persistence and you lose all your devtools state.